pages/common/traefik.md

```diff
@@ -0,0 +1,19 @@
+# traefik
+
+> A HTTP reverse proxy and load balancer.
+
+- Start server with default config:
```

Comment:
Perhaps making it "traefik server" instead of just server? Same with below.

Reply:
I was merely matching the same wording used on the similar nginx page. I can adjust it if you'd like...

Reply:
Ah, ok! Let's leave it then :smiley_cat: